.github/workflows/centos_7.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           # Our testing expects that the init process (PID 1) will
           # reap orphan processes. At least the following test leans
           # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
-          PACKPACK_EXTRA_DOCKER_RUN_PARAMS: '--init'
+          PACKPACK_EXTRA_DOCKER_RUN_PARAMS: '--init --memory=7G --memory-swap=7G'
```

Comment:
I would say that `sudo swapoff -a` is much more obvious solution. Are there reasons to don't use it?

Reply:
I see, we run the whole job in several workflows like `release`. Okay. However I anyway against hardcoding of a memory value. Can we do something like the following?

```sh
TOTAL_MEMORY=$(cat /proc/meminfo | grep MemTotal | awk '{ print $2; }')
```

Reply:
Unfortunately we still use docker containers in workflows and it denies the ability to run shell command before it. After resolve of tarantool/tarantool-qa#101 issue we'll have this ability and will change it to some simple command.

Reply:
We still can use `sudo swapoff -a` in workflows that does not use `container:` property, right? Maybe it worth to do so? And we can avoid hardcoding in `.travis.mk`.

Reply:
Right, I'll do it.

Reply:
I've missed, that the simplest way is just to use `--memory-swappiness=0` option for docker, as suggested here:
https://docs.docker.com/config/containers/resource_constraints/#--memory-swappiness-details

Reply:
> A value of 0 turns off anonymous page swapping.

I read it as 'file caches will be swapped anyway'.

Reply:
Right, so let's do it in ways as it possible for now.